_includes/delete-token.md

```diff
@@ -0,0 +1,44 @@
+{% assign documentation_section = include.documentation_section %}
```

Comment:
Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6038721)